app/models/invenio_rdm_record_converter.rb

```diff
@@ -174,14 +174,14 @@ def resource_type(digitalhub_subtype)
   def contributors(contributors)
     contributors.map do |contributor|
       contributor_json = build_creator_contributor_json(contributor)
-      contributor_json[:person_or_org].merge!({"role": ROLE_OTHER})
+      contributor_json.merge!({"role": {}})
       contributor_json
     end
   end
 
   def build_creator_contributor_json(creator)
     if creator_data = @@person_or_org_data[creator]
-      return creator_data
+      return creator_data.merge({"role": {}, "affiliations": {}})
```

Comment:
Simple way to meet the requirements. Role is not determined yet and affiliations are all nil for production DigitalHub.

If it's not clear, I moved all the previous data we had here (except for role and affiliations) into the person_or_org field. So, really, all that's happened is that the previous information is now the value for the key "person_or_org" and the other two key-value pairs just hold empty hashes.